shell/artifacts/Products/ShowProduct.manifest

```diff
@@ -12,7 +12,5 @@ particle ShowProduct in 'source/ShowProduct.js'
   ShowProduct(in Product product)
   affordance dom
   affordance dom-touch
-  consume root
-    provide annotation
-      view product
-  description `show product`
+  consume item
+  //description `show product`
```

Comment:
Intended to leave disable?

Reply:
it is, added a comment and a TODO to uncomment it, when nested recipes are further along.